src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSPackageRestoreManager.cs

```diff
@@ -70,6 +70,7 @@ private void OnNuGetProjectAdded(object sender, NuGetProjectEventArgs e)
                 await TaskScheduler.Default;
 
                 await RaisePackagesMissingEventForSolutionAsync(solutionDirectory, CancellationToken.None);
+                await RaiseAssetsFileMissingEventForSolutionAsync(solutionDirectory, e.NuGetProject.ToString(), CancellationToken.None);
```

Comment:
Can you elaborate why we need this/ 

The OnNugetProjectAdded is called a lot, and unless the PM UI is open, which it can't be for a project that's not loaded, this event won't have a listener I think.

Reply:
Deleting this because the banner will be only shown in project level